irods_runner.Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM ubuntu:22.04
+
+SHELL ["/bin/bash", "-c"]
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Re-enable apt caching for RUN --mount
+RUN rm -f /etc/apt/apt.conf.d/docker-clean && \
+    echo 'Binary::apt::APT::Keep-Downloaded-Packages "true";' > /etc/apt/apt.conf.d/keep-cache
+
+# Make sure we're starting with an up-to-date image
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
+    --mount=type=cache,target=/var/lib/apt,sharing=locked \
+    apt-get update && \
+    apt-get upgrade -y && \
+    apt-get autoremove -y --purge && \
+    rm -rf /tmp/*
+# To mark all installed packages as manually installed:
+#apt-mark showauto | xargs -r apt-mark manual
+
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
+    --mount=type=cache,target=/var/lib/apt,sharing=locked \
+    apt-get update && \
+    apt-get install -y \
+        gnupg \
+        lsb-release \
+        wget \
+    && \
+    rm -rf /tmp/*
+
+RUN wget -qO - https://packages.irods.org/irods-signing-key.asc | apt-key add - && \
+    echo "deb [arch=amd64] https://packages.irods.org/apt/ $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/renci-irods.list
+
+ARG irods_version=4.3.1-0~jammy
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
+    --mount=type=cache,target=/var/lib/apt,sharing=locked \
+    apt-get update && \
+    apt-get install -y \
+        apt-transport-https \
+        g++-11 \
+        gcc \
+        gcc-11 \
+        irods-runtime=${irods_version} \
+        irods-externals-clang-runtime13.0.0-0 \
+        irods-externals-jwt-cpp0.6.99.0-0 \
+    && \
+    rm -rf /tmp/*
+
+COPY /*.deb /
+
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
+    --mount=type=cache,target=/var/lib/apt,sharing=locked \
+    apt-get update && \
+    apt-get install -y /*.deb && \
```

Comment:
Does installing local packages with `apt-get` actually work now?

Reply:
Yes, since apt 1.1 says the release notes...

https://mvogt.wordpress.com/2015/11/30/apt-1-1-released/

Reply:
Oh, you said `apt-get`... not sure on that.

Reply:
It appears to work. Everything here results in the expected behavior.

Reply:
Done.